File: dotnet/src/MissionControl.Client/MissionControlHeaders.cs
namespace MissionControl.Client;

/// <summary>
/// Header names used by the MissionControl Partner API.
/// </summary>
public static class MissionControlHeaders
{
    /// <summary>
    /// Identifies the partner on whose behalf the request is made.
    /// This value is provided via the portal on the client credentials page.
    /// </summary>
    public const string PartnerId = "X-Partner-Id";
}

File: typescript/src/client/missionControlHeaders.ts
/**
 * Header names used by the MissionControl Partner API.
 */
export const MissionControlHeaders = {
  /**
   * Identifies the partner on whose behalf the request is made.
   * This value is provided via the portal on the client credentials page.
   */
  PartnerId: "X-Partner-Id",
} as const;
